packages/flutter/test/cupertino/adaptive_text_selection_toolbar_test.dart

```diff
@@ -30,6 +30,11 @@ void main() {
     );
   });
 
+  Finder findOverflowNextButton() => find.byWidgetPredicate((Widget widget) =>
+  widget is CustomPaint &&
+      '${widget.painter?.runtimeType}' == '_RightCupertinoChevronPainter',
+  );
```

Comment:
Nit: The multiple inline function `=>` things are a little bit confusing, maybe turn one or both into regular `() {}` functions?

Or at least indent lines 34 and 35 I think.